src/main/java/com/google/sps/servlets/SchedulingTest.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Google LLC
```

Comment:
I think this should be in the `src/main/javatests/com/google` folder

Reply:
Do we have that folder created yet? The tests in the walkthroughs for the start project were inside src/test/com/google/sps. If that hasn't been created yet, I can go ahead and do that.

Reply:
I don't think so. I made one in my homepage branch separately but that might cause some conflicts. Yeah, can you try creating it in master?